docs/transforms.md

```diff
@@ -473,6 +473,20 @@ Scales the number by 16 (default web font size) and adds 'px' to the end.
 ```
 
 
+* * *
+
+### size/pxToRem
```

Comment:
Just checking, did you write this by hand? This markdown file is generated by a script that takes the JSDoc content and formats it into markdown.

Reply:
Oops, didn't realize it was auto generated. Removing...